hack/go_container.sh

```diff
@@ -82,6 +82,8 @@ run_in_go_container() {
       -e GO111MODULE -e GOPROXY -e CGO_ENABLED -e GOOS -e GOARCH \
     `# pass through proxy settings` \
       -e HTTP_PROXY -e HTTPS_PROXY -e NO_PROXY \
+    `# use host certificates` \
+      -v "/etc/ssl/certs:/etc/ssl/certs" \
```

Comment:
I think this is OS-specific, not all distros have the  certs in the same folder IIRC
Also, I think  the mount should be read-only to avoid possible issues

Reply:
maybe adding it as an env variable? CERT_DIR?

Reply:
the OS specific ones may not be bind mountable anyhow (eg on mac)

It might be best to use a custom `GOIMAGE` instead. I'm not sure if there's a "best practice" for this though.